docs/playlist/indie.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebarDepth: 2
+title: Indie
+---
+
+# Indie Playlist Music
+
+| Playlist Name | Playlist Link | Platform | Playlist Author Name/Contributor Name | Note | Subgenre |
+| ------------- |-------------- | -------- | ------------------------------------- | ---- | -------- |
+| Indie Indonesia 2021 | [Spotify](https://open.spotify.com/playlist/5oK4mEz7aDf1Uv6zaMRx7g?si=876bf23de81c410c) | Spotify | [Fatur](https://open.spotify.com/user/z3alt7fldts933icoqkcxqgu4?si=bfda747fc2ad4269) | - | - |
+| Indie Pop | [Spotify](https://open.spotify.com/playlist/37i9dQZF1DWWEcRhUVtL8n?si=3ad0602da4544d04) | Spotify | [Spotify](https://open.spotify.com/user/spotify?si=a4f4cbbdf60f44a1) | - | - |
+| Indie Folk 2021 | [Spotify](https://open.spotify.com/playlist/1Qs40FRP061tVZ9x2npAE3?si=ae0eddc6e9d34b7c) | Spotify | [Indie & Folk Radio](https://open.spotify.com/user/indiefolkradio?si=eaeb015c35b14b6e) | - | - |
+| IndieNesia | [Spotify](https://open.spotify.com/playlist/37i9dQZF1DXd82NU5rAcTZ?si=97efd86cf078468c) | Spotify | [Spotify](https://open.spotify.com/user/spotify?si=241cbea1bc2e4a73) | - | - |
+| Indie Chillout | [Spotify](https://open.spotify.com/playlist/37i9dQZF1DX9B1hu73DioC?si=f4494cc45c6c40fb) | Spotify | [Spotify](https://open.spotify.com/user/spotify?si=a0d374b81cd34560) | - | - |
```

Comment:
Please sort alphabetically, refer to this https://awesome-music-playlist.vercel.app/contribution.html#add-new-playlist